src/utils.js

```diff
@@ -30,4 +30,20 @@ const unquoteString = string => {
   return string;
 };
 
-module.exports = { reduceCSSSelector, unquoteString };
+/**
+ * Convert ConosleMessage to array of values which were passed to console.
+ *
+ * @param {ConosleMessage} msg
+ * @returns {Promise<Array<any>>}
+ */
+const consoleMessageToArguments = (msg) =>{
```

Comment:
Perhaps your `prettier` isn't set up but I bet Travis will complain about this lacking whitespace before the `{`.